jsonld/prefixes.context.jsonld

```diff
@@ -1,14 +1,25 @@
 {
-   "_comments": "Auto generated from prefixes.yaml by jsonldcontextgen.py version: 0.1.1\nGeneration date: 2021-01-26 16:13\nSchema: prefixes\n\nid: https://ccdh.org/model/prefixes\ndescription: \nlicense: https://creativecommons.org/publicdomain/zero/1.0/\n",
+   "_comments": "Auto generated from prefixes.yaml by jsonldcontextgen.py version: 0.1.1\n    Generation date: 2021-04-27 00:07\n    Schema: prefixes\n    \n    id: https://ccdh.org/model/prefixes\n    description: \n    license: https://creativecommons.org/publicdomain/zero/1.0/\n    ",
    "@context": {
-      "type": "@type",
       "ADM": "https://ccdh.org/models/ADM/",
       "BRIDG": "https://fill.me.in/BRIDG/",
       "FHIR": "http://hl7.org/fhir/",
-      "GDC": "http://fill.me.in/GDC",
-      "HTAN": "http://fill.me.in/ICDC",
-      "ICDC": "http://fill.me.in/ICDC",
-      "PDC": "http://fill.me.in/PDC",
+      "GDC": {
+         "@id": "http://fill.me.in/GDC",
+         "@prefix": true
+      },
+      "HTAN": {
+         "@id": "http://fill.me.in/ICDC",
+         "@prefix": true
+      },
+      "ICDC": {
+         "@id": "http://fill.me.in/ICDC",
+         "@prefix": true
+      },
+      "PDC": {
+         "@id": "http://fill.me.in/PDC",
+         "@prefix": true
+      },
```

Comment:
These would not have become objects if the expansions had ended in a `/`, like for `BRIDG` above.

Reply:
I've added trailing `/`s to them in ca1c1b891f0e3, and regenerated the files in f2b54dd79512.